### DISCUSSION TITLE
Leechblock stops working

### DISCUSSION THREAD

**Discussion by Tor-orme** [proginosko/LeechBlockNG#462] [Q&A] [answered]
I used Leechblock for some years until it suddenly it stopped blocking sites, so I stopped using it.
I tried again a few years later and successfully blocked sites, and also blocked access to Options.
But when I tried to add a new site to the Options with the Add Sites function but it did not block it
So I removed LB from my Firefox extensions, re-downloaded it and added the new site in with the others. Then I blocked access to Options from 0300-0200.  Now Now it doesn’t block anything.
Ive tried rebooting the PC each time I do these changes.
Im using an Android desktop PC with Firefox and Microsoft Office.
Ive got limited vision and limited computer skills.  Any help[ gratefully received.

**Comment by proginosko** [comment, marked as answer]
If you want to block for a period that goes through midnight, you need to specify it like this: 0000-0200,0300-2400

More explanation here: https://www.proginosko.com/leechblock/faq/overnight-block/

**Comment by Tor-orme** [comment]
MANY THANKS!   I think Ive finally got it running OK.   I appreciate your replying so fast.
Warm regards
Torfrida
